index.d.ts

```diff
@@ -254,9 +254,14 @@ declare namespace execa {
 		killed: boolean;
 
 		/**
-		The signal that was used to terminate the process.
+		The name of the signal that was used to terminate the process.
```

Comment:
Unrelated to this PR, but I just noticed, when something can be `undefined`, we should try to describe in what scenarios it's defined vs undefined. This can be a big help for the user in knowing when it will be used.